src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -352,6 +355,17 @@ private VariableDesignationSyntax Transform( VariableDesignationSyntax des )
 			_ => UnhandledSyntax( des )
 		};
 
+	private SeparatedSyntaxList<CollectionElementSyntax> Transform(
+		SeparatedSyntaxList<CollectionElementSyntax> original
+	) => SyntaxFactory.SeparatedList(
+		original.Select( elem => elem switch {
+			ExpressionElementSyntax ee => ee.WithExpression( Transform( ee.Expression ) ),
+			SpreadElementSyntax sp => sp.WithExpression( Transform( sp.Expression ) ),
+			_ => UnhandledSyntax( elem )
+		} ),
+		original.GetSeparators()
+	);

```

Comment:
Actually part of this is already in place with the TransformAll combinator.
It doesn't do original.GetSeparators(), though.

I'll do a follow-up PR to switch to that, but also add the separators. That will incur some noise because we will have to fix up the whitespace in unrelated tests.